gsoc-2022-ideas.md

```diff
@@ -269,3 +272,56 @@ Medium: The task does require more than just Bash to Python script conversion. I
 _Duration_: 175 hours or 350 hours
 
 _Mentor_: [@mkurc-ant](https://github.com/mkurc-ant)
+
+
+## Symmetrical placement and routing APIs in OpenFASOC
+
+[OpenFASOC](https://github.com/idea-fasoc/OpenFASOC) is a framework used for automated IC design generation. It sits on top of tools such as [OpenROAD](https://github.com/The-OpenROAD-Project/OpenROAD) primararly but recently is using tools such as [gdsfactory](https://github.com/gdsfactory/gdsfactory) and [ALIGN](https://github.com/ALIGN-analoglayout/ALIGN-public). Currently, we have enabled a few Python based functions that allow us to help with our analog layout requirements, and we are planning to create more general APIs that could be generalized to new analog designs.
```

Comment:
@msaligane do you think this could also include work on https://github.com/idea-fasoc/OpenFASOC/issues/38 ?